### ISSUE TITLE
error installation

### ISSUE THREAD

**Issue by arademaker** [own-pt/delphin-rdf#16]
@yfaria can you help?

```
(venv) ar@tenis delphin.rdf % pip install .
Processing /Users/ar/hpsg/delphin.rdf
  DEPRECATION: A future pip version will change local packages to be built in-place without first copying to a temporary directory. We recommend you use --use-feature=in-tree-build to test your packages with this new behavior before it becomes the default.
   pip 21.3 will remove support for this functionality. You can find discussion regarding this at https://github.com/pypa/pip/issues/7555.
    ERROR: Command errored out with exit status 1:
     command: /Users/ar/venv/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-req-build-3dxddjin/setup.py'"'"'; __file__='"'"'/private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-req-build-3dxddjin/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk
         cwd: /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-req-build-3dxddjin/
    Complete output (10 lines):
    running egg_info
    creating /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk/Delphin_RDF.egg-info
    writing /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk/Delphin_RDF.egg-info/PKG-INFO
    writing dependency_links to /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk/Delphin_RDF.egg-info/dependency_links.txt
    writing requirements to /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk/Delphin_RDF.egg-info/requires.txt
    writing top-level names to /private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk/Delphin_RDF.egg-info/top_level.txt
    writing manifest file '/private/var/folders/b_/7nbv248s2nq019mcx58xrrb80000gn/T/pip-pip-egg-info-qqyop0sk/Delphin_RDF.egg-info/SOURCES.txt'
    package init file 'delphin/__init__.py' not found (or not a regular file)
    package init file 'delphin/cli/__init__.py' not found (or not a regular file)
    error: package directory 'delphin/codecs' does not exist
    ----------------------------------------
WARNING: Discarding file:///Users/ar/hpsg/delphin.rdf. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by FredsoNerd** [comment]
The `codecs` directory was removed in some recent updates, but not removed from the `setup.py`. I've just removed `delphin.codecs` from setup; everything in working well.

**Comment by arademaker** [comment]
```
 pip install .
Processing /Users/ar/hpsg/delphin-rdf
  DEPRECATION: A future pip version will change local packages to be built in-place without first copying to a temporary directory. We recommend you use --use-feature=in-tree-build to test your packages with this new behavior before it becomes the default.
   pip 21.3 will remove support for this functionality. You can find discussion regarding this at https://github.com/pypa/pip/issues/7555.
Requirement already satisfied: pydelphin in /Users/ar/venv/lib/python3.8/site-packages (from Delphin-RDF==1.0.0) (1.5.1)
Requirement already satisfied: rdflib in /Users/ar/venv/lib/python3.8/site-packages (from Delphin-RDF==1.0.0) (5.0.0)
Requirement already satisfied: progress==1.5 in /Users/ar/venv/lib/python3.8/site-packages (from pydelphin->Delphin-RDF==1.0.0) (1.5)
Requirement already satisfied: penman==1.1.0 in /Users/ar/venv/lib/python3.8/site-packages (from pydelphin->Delphin-RDF==1.0.0) (1.1.0)
Requirement already satisfied: six in /Users/ar/venv/lib/python3.8/site-packages (from rdflib->Delphin-RDF==1.0.0) (1.16.0)
Requirement already satisfied: isodate in /Users/ar/venv/lib/python3.8/site-packages (from rdflib->Delphin-RDF==1.0.0) (0.6.0)
Requirement already satisfied: pyparsing in /Users/ar/venv/lib/python3.8/site-packages (from rdflib->Delphin-RDF==1.0.0) (2.4.7)
Using legacy 'setup.py install' for Delphin-RDF, since package 'wheel' is not installed.
Installing collected packages: Delphin-RDF
    Running setup.py install for Delphin-RDF ... done
Successfully installed Delphin-RDF-1.0.0
```

The problem was solved by... 

1. in the future, let us ALWAYS make a PR to the master
2. what is the warning ? We need to document the new way to install packages

**Comment by yfaria** [comment]
There was a reason for this codecs folder; I believe that it was to be formally considered a codecs in PyDelphin standards but I can't remember. We decided to face this problem later, so the branch [`codecs`](https://github.com/own-pt/delphin-rdf/tree/codecs) was created for that and we removed the folder about it at master to make the repo cleaner. The problem was that we forgot to remove this on the installation file, but @FredsoNerd commit solved that. Installing now is the same way described in the README, `pip install /path/to/delphin-rdf` with the optional `-e` to install in the developer mode
